src/test/java/com/mycompany/selenium/Task_1.java

```diff
@@ -0,0 +1,84 @@
+package com.mycompany.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.function.Function;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by ekaterina.kazinets on 10/24/2017.
+ */
+public class Task_1 {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/chromedriver-v2.30-win32/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("http://ss.com");
+    }
+
+    @Test
+    public void testChangeLanguage() throws Exception {
+        WebElement lang = driver.findElement(By.cssSelector(".menu_lang .a_menu"));
+        if (!lang.getText().equals("RU")) {
+            lang.click();
+
+        }
+        Assert.assertEquals(lang.getText(), "RU");
+        Assert.assertEquals(lang.getAttribute("title"), "По-русски");
+
+    }
+
+    @Test
+    public void testSimpleSearch() {
+        WebElement lang = driver.findElement(By.cssSelector(".menu_lang .a_menu"));
```

Comment:
try to name elements post/prefixed with element type (e.g. - `switchLanguageLink`), so that it would be instantly clear what is the element and how to interact with it in the test 
In addition, try to avoid variable shortenings